src/Protocol/ActivityPub/Transmitter.php

```diff
@@ -480,6 +481,27 @@ private static function getTypeOfItem($item)
 		return $type;
 	}
 
+	/**
+	 * @brief Creates the activity or fetches it from the cache
```

Comment:
If the function description is a one-liner, I don't think you need the non-standard `@brief` docblock variable. It stunts some documentation parsers that thinks there's no description at all. `@brief` should be use in addition to a regular description (without docblock variable in front of it).

Reply:
What exactly should I do now? I used the ```@brief``` oneliner for **every** function in the new files. So removing it just at this single place doesn't make much sense.

Reply:
Fair enough.